client/state/data-layer/wpcom/sites/activity/schema.json

```diff
@@ -0,0 +1,88 @@
+{
+	"additionalProperties": false,
+	"type": "object",
+	"required": [ "current", "itemsPerPage", "page", "totalItems", "totalPages" ],
+	"properties": {
+		"@context": { "type": "string" },
+		"first": { "type": "string" },
+		"id": { "type": "string" },
+		"itemsPerPage": { "type": "integer" },
+		"last": { "type": "string" },
+		"page": { "type": "integer" },
+		"summary": { "type": "string" },
+		"totalItems": { "type": "integer" },
+		"totalPages": { "type": "integer" },
+		"type": { "type": "string" },
+		"current": {
+			"additionalProperties": false,
+			"type": "object",
+			"required": [ "orderedItems", "totalItems" ],
+			"properties": {
+				"id": { "type": "string" },
+				"next": { "type": "string" },
+				"prev": { "type": "string" },
+				"orderedItems": {
+					"type": "array",
+					"items": { "$ref": "#/definitions/item" }
```

Comment:
At some point in the future, this could become `oneOf` with schemas for all the different items. Of course, that might be overkill 😛